qa-tests-backend/e2e-early-lock.sh

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env bash
```

Comment:
This file will not be merged into master. However, it might be merged into the parent branch. This script might serve as the basis for e2e tests.